docs/application/native/guides/text-input/voice-control.md

```diff
@@ -1,4 +1,6 @@
 # Voice Control
+[!NOTE]
+All Voice Control Elementary APIs are deprecated since Tizen 10.0 and will be removed after two releases without any alternatives.
```

Comment:
```suggestion
> [!NOTE]
> All Voice Control Elementary APIs are deprecated since Tizen 10.0 and will be removed after two releases without any alternatives.
```